Client/game_sa/CVehicleSA.h

```diff
@@ -315,6 +315,29 @@ struct CTrainFlags
     unsigned char unknown7 : 8;
 };
 
+struct CTransmissionGear
+{
+    float maxGearVelocity;
+    float changeUpVelocity;
+    float changeDownVelocity;
+};
+
+struct CTransmission
+{
+    CTransmissionGear gears[6];
+    byte              driveType;
+    byte              engineType;
+    byte              numOfGears;
+    byte              pad;
+    DWORD             flags;
+    float             engineAcceleration;
+    float             engineInertia;
+    float             maxVelocity;
+    byte              pad2[4];
+    float             minVelocity;
+    float             curVelocity;

```

Comment:
```suggestion
    float             curSpeed;
```

Do you know why gta_sa defines this as "speed" rather than "velocity"